### ISSUE TITLE
Client-side-generated .xml file: folder duplication in image file paths 

### ISSUE THREAD

**Issue by TacticBlack** [PolarBean/DeepSlice#9]
As I was testing different things in for a [previous issue](https://github.com/PolarBean/DeepSlice/issues/8), I noticed that the client-side-generated .xml files were the ones that did not show the images in QuickNII. If you look at the file path in the top of the window of image 2 below and compare it to the same in image 1 you will see it too.

**Image1** (Online-Generated)
![OnlineJPGAlignment](https://user-images.githubusercontent.com/79601721/166712133-6b5f5f86-be11-4afb-9cc9-46456130f6d5.JPG)

**Image2** (Client-Side-Generated)
![ClientSideJPGAlignment4](https://user-images.githubusercontent.com/79601721/166712148-30e8f72f-13ed-49b7-b392-355966c80b61.JPG)

The "ch1_DAPI" folder is listed twice in the path for the one generated on the client side. Consequently, the images dont get loaded into QuickNII for comparison to the atlas. I originally saved the output of the model to the parent folder ("Images") and tried saving it to the child folder with the images to no avail. The image file paths remained containing the duplicate folder. I can open a separate bug report for this if you'd like.

**Comment by PolarBean** [comment]
Yes this is a problem with my use of keras ImageDataGenerator, this is solved in an alpha version of DeepSlice that will be released in the coming months. But for now the solution is to place the XML outside the folder when using the python repo. this is covered in the guide but is confusing given the differing behaviour of web vs python.
